.changeset/new-toys-occur.md

```diff
@@ -0,0 +1,6 @@
+---
+"web": patch
```

Comment:
```suggestion
```

Reply:
thank you! It's my first time using changeset and I wasn't sure whatś wrong..